From ad7ff5ec90c6f5026e99698aa264f959478b5059 Mon Sep 17 00:00:00 2001
From: Chetan <chetansky98@gmail.com>
Date: Wed, 21 Jul 2021 18:35:17 +0530
Subject: [PATCH] Second test Commit

---
 File2.py | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 File2.py

diff --git a/File2.py b/File2.py
new file mode 100644
index 0000000..3a82276
--- /dev/null
+++ b/File2.py
@@ -0,0 +1 @@
+print(27/2 * 124)
-- 
2.32.0.windows.2

